kubecost/cost-analyzer-helm-chart
# Ajay tripathy etl readonly perms

## What does this PR change?
Flag support for running ETL in a low-permission mode where it simply connects to a PromQL database assumed to have cost data and its own configmaps.


## Does this PR rely on any other PRs?
The supporting PRs are already merged

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
Users will be able to run kubecost without permissions outside of its own namespace, assuming agents in other namespaces have those permissions.


## Links to Issues or ZD tickets this PR addresses or fixes

- 
- 


## How was this PR tested?
1. Run with the new flag disabled, and ensure the right metrics are generated
2. Run with the new flag enabled, ensure kubecost runs and generates metric data

## Have you made an update to documentation?
The flag should really be used sparingly; not suitable for almost every user.
